core/nodejsAction/Dockerfile

```diff
@@ -52,6 +52,7 @@ swagger-tools@0.8.7 \
 tmp@0.0.28 \
 watson-developer-cloud@1.4.1 \
 when@3.7.3 \
+ws@1.1.0 \
```

Comment:
will require updating `reference.md` and generating pr to bluemix docs @csantanapr.